docs/getting_started/Datus_tutorial.md

```
@@ -142,18 +156,18 @@ Output:
+datus-cli --datasource california_schools
 /subject
 ```
 
 ![Subject Tree Structure](../assets/tutorial_subject_tree.png)
 
-### Step [5/5] Build Subagents
+### Step [6/6] Build Subagents
 
 The tutorial automatically generates two [subagents](../subagent/introduction.md):
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)